.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,18 @@
+name: Request a new feature
+description: New demands? Tell us what Pelorus could do to help you
+labels: ["kind/feature", "feature"]
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for using Pelorus and helping it always improve!
+  - type: textarea
+    attributes:
+      label: Feature description
+      description: |
+        Tell us what new functionality would you like Pelorus to have. The more
+        info you provide, the better we can address it
+      placeholder: |
+        Fell free to add images and other files that can help us understand the
+        feature
+    validations:
```

Comment:
Is there a way to give an importance checkbox or label? E.g. "how badly do you need this feature and what are expected timeframes"

Also would be good to ask how the Pelorus is used so we understand the implementation implications for the feature.

Reply:
The first one should help us prioritize our work.

Second one is to understand if the user have for example planned work-arounds if we don't deliver such feature, or it's a feature that will prevent user from using Pelorus all together.

Reply:
First one being a dropdown with options priority 1, 2, etc, right? (please give me the name of the options, we have a lot of `priority` labels for example)

alright, understood the second one as well, going to right it down

Reply:
what about

- ⏳ Not a priority, it does not block me of using Pelorus
- ⌛ Soon, it will greatly improve my experience using Pelorus
- 🚨 As soon as possible, it block me of using Pelorus

Reply:
I like it, just reword:

```
Low priority, it does not block me of using Pelorus
It will greatly improve my experience using Pelorus
As soon as possible, it blocks me of using Pelorus
```